packages/arb-token-bridge-ui/src/components/TransactionHistory/helpers.ts

```diff
@@ -406,6 +406,11 @@ export async function getUpdatedCctpTransfer(
   }
 
   const receipt = await getTxReceipt(tx)
+
+  if (!receipt) {
```

Comment:
We add a raw pending tx so it's possible it won't have a receipt yet. In this case we keep the tx as is until we get the receipt.